<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>V2N75SetupDialog</class>
 <widget class="QDialog" name="V2N75SetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>V2 N75 Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="n75TableGroupBox">
     <property name="title">
      <string>N75 maps</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="n75comboBox">
           <item>
            <property name="text">
             <string>Low Boost</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High Boost</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="n75ReadPushButton">
           <property name="text">
            <string>Read from MD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="n75WritePushButton">
           <property name="text">
            <string>Write to MD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loadEepromPushButton">
           <property name="text">
            <string>Load from EEprom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="writeEepromPushButton">
           <property name="text">
            <string>Write to EEprom</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>V2N75SetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>V2N75SetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
